pkg/api/api.go

```diff
@@ -32,3 +33,8 @@ type PortDriverInfo struct {
 	Protos                  []string `json:"protos"`
 	DisallowLoopbackChildIP bool     `json:"disallowLoopbackChildIP,omitempty"` // since API v1.1.1
 }
+
+type ChildDriverInfo struct {
+	ConfiguresInterface	bool     `json:"configuresInterface"`
+}
```

Comment:
I don't think that this has to be published to the REST API

Reply:
Right, I also wanted to avoid that, but I have no idea where else to define this in a way that can be used by all the drivers.